documentation/package-lock.json

```diff
@@ -1,8 +1,1276 @@
 {
```

Comment:
Can this file also be removed?

Reply:
Technically, I think so.  It will be generated every time anyone installs the iota-wiki-cli though.  If you prefer that I could add it to the .gitignore.  Let me know what you prefer.

Reply:
Could also stay, but I guess we get dependabot PRs then, should we just merge them always then? I don't want to test them every time locally

Reply:
I'll delete it and add it to the gitignore then.  The dependabot warnings don't make any sense as the wiki repo is the one actually building.